src/main/java/seedu/wife/logic/commands/foodcommands/ViewCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.wife.logic.commands.foodcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.wife.commons.core.Messages;
+import seedu.wife.commons.core.index.Index;
+import seedu.wife.logic.commands.Command;
+import seedu.wife.logic.commands.CommandResult;
+import seedu.wife.logic.commands.exceptions.CommandException;
+import seedu.wife.model.Model;
+import seedu.wife.model.food.Food;
+
+/**
+ * Views the food identified by the index number used in the displayed food list.
+ */
+public class ViewCommand extends Command {
+
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Views the food identified by the index number used in the displayed food list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_VIEW_FOOD_SUCCESS = "Viewed Food: %1$s";
+
+    private final Index targetIndex;
+
+    public ViewCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
```

Comment:
Should you have a `requireNonNull` here to make sure the inserted `targetIndex` is not a null value?